Program.cs

```diff
@@ -11,22 +11,17 @@ internal class Program
         static void Main(string[] args)
         {
 
-            try
-            {
-                Console.WriteLine("Hello");
-            }
-            catch (ArgumentNullException)
-            {
-                Console.WriteLine("A");
-            }
-            catch (Exception)
-            {
-                Console.WriteLine("B");
-            }
-            finally
-            {
-                Console.WriteLine("C");
-            }
+           int n1, n2, resultado;
+
+            Console.WriteLine("Digite qualquer número: ");
+            n1 = int.Parse(Console.ReadLine());
```

Comment:
Tratar cenário em que o usuário insira um valor diferente de número.
Ao invés de usar `int.Parse`, utilize `int.TryParse` que vai garantir que o programa não quebre quando houver uma entrada inválida por parte do usuário.
```suggestion
            int.TryParse(Console.ReadLine(), out n1);
```